src/PkgCacheInspector.jl

```diff
@@ -182,6 +225,22 @@ function count_module_specializations(new_specializations)
     return modcount
 end
 
+# count_internal_specializations is defined in MethodAnalysisExt when MethodAnalysis is loaded
+count_internal_specializations(::Any) = nothing
+
+# Count the number of methods defined within each of the package's own modules.
+# These are methods that belong to the modules stored in the package image,
+# as opposed to external methods which extend functions from other modules.
+function count_internal_methods(info::PkgCacheInfo)
+    method_counts = Dict{Module,Int}()
+    Base.visit(Core.GlobalMethods) do method
+        if method.module in info.modules
+            method_counts[method.module] = get(method_counts, method.module, 0) + 1
+        end
+    end
```

Comment:
It's fun living on the bleeding edge https://github.com/JuliaLang/julia/pull/59158